projects/core/src/lib/components/info-panel/info-panel.component.ts

```diff
@@ -0,0 +1,28 @@
+import {
+	ChangeDetectionStrategy,
+	Component,
+	EventEmitter,
+	OnInit,
+	Output,
+	ViewEncapsulation
+} from "@angular/core";
+
+@Component({
+	selector: "jsmu-info-panel",
+	templateUrl: "./info-panel.component.html",
+	styleUrls: ["./info-panel.component.scss"],
+	encapsulation: ViewEncapsulation.None,
```

Comment:
using ViewEncapsulation.None is not recommended as it can affect other components

Reply:
done